docker-compose.override.yml

```diff
@@ -29,6 +29,7 @@ services:
   app:
     container_name: meditor_app
     build: packages/app
+    restart: on-failure:3
```

Comment:
Should we limit it to 3? Or just let it keep restarting?

Reply:
I limited it to 3 just in case the container gets stuck on a restart loop. We can change that to just restart.

Reply:
Resolved in [36b4556](https://github.com/nasa/gesdisc-meditor/pull/47/commits/36b45564a2a0b939f94777bbfdec0ac7ee4e62a7).